extensions/amazon-aws/src/queues.tsx

```diff
@@ -1,11 +1,11 @@
 import { getPreferenceValues, ActionPanel, List, Detail, Action, confirmAlert, Toast, showToast } from "@raycast/api";
 import { useCachedPromise } from "@raycast/utils";
+import { chunk } from "lodash";
```

Comment:
let's use direct import instead to prevent loading full lodash module

```suggestion
import chunk from "lodash/chunk";
```

Reply:
yup, good point, done.